test/fixtures/gomod/big/go.mod

```diff
@@ -195,6 +195,8 @@ require (
 	vbom.ml/util v0.0.0-20160121211510-db5cfe13f5cc
 )
 
+replace k8s.io/api => ./staging/src/k8s.io/api
```

Comment:
I moved this to here in order to check the 2 possible ways of the [replace directive](https://golang.org/ref/mod#go-mod-file-replace)